packages/neo-one-server-client/src/ServerManager.ts

```diff
@@ -91,6 +74,54 @@ export class ServerManager {
     }
   }
 
+  public async getServerVersion(): Promise<string | undefined> {
+    let versionContents: string | undefined;
+    try {
+      versionContents = await fs.readFile(this.versionPath, 'utf8');
+    } catch (error) {
+      if (error.code === 'ENOENT') {
+        return undefined;
+      }
+      throw error;
+    }
+
+    return versionContents;
+  }
+
+  public async getCurrentVersion({ port }: { readonly port: number }): Promise<string | undefined> {
+    const client = new Client({ port });
+    const startTime = utils.nowSeconds();
+    // tslint:disable-next-line no-loop-statement
+    while (utils.nowSeconds() - startTime <= 5) {
+      try {
+        return client.getVersion();
```

Comment:
This needs to actually `await` the returned promise in order to catch errors. I.e.
```
const version = await client.getVersion();

return version;
```

and then ignore the tslint error.

Reply:
👍